src/engraving/dom/note.cpp

```diff
@@ -2276,7 +2276,9 @@ int Note::line() const
 void Note::setString(int val)
 {
     m_string = val;
-    mutldata()->setPosY(m_string * spatium() * 1.5);
+    if (onTabStaff()) {
+        mutldata()->setPosY(m_string * spatium() * 1.5);
```

Comment:
@miiizen I think it might be worth investigating whether this line can be removed completely. It seems a bit strange that a simple setter for `m_string` would mess with some graphical data that should be calculated exclusively during layout. 
Also, the hard-coded `1.5` really looks like someone actually meant `staffTypeForElement(this)->lineDistance()`...

I've tried removing this line locally, and it didn't seem to cause problems at first sight.

Interestingly, this line is very old; it even existed in some form in the initial commit... https://github.com/musescore/MuseScore/commit/6047361bd08877924d1ea2468075bc97f8a9ee4a#diff-42559676faac26b3fc535d6820087e705bf7f96d6dbcd42de1206ac5b8943d57R1476

Reply:
I've had another look and this is indeed calcuated more thoroughly in `ChordLayout::layoutTablature`.  We can remove it completely.